CYOAfeb21

```diff
@@ -0,0 +1,225 @@
+	def position(self, value):
+		self.value= value
+#Items			
+	def items(self):
+		return self.items
+	def items(self, value):
+		self.value= value
+###//Define NPC Subclass//###
+###Not needed since they only need character parameters###
+
+###Start NPC Bob###
+bob= Character('Bob', 'Town', 'Main Key')
+#Function for printing bob.
+def bobsheet():
+	print(f"""
+	Name: {bob.name}
+	Position: {bob.position}
+	Items: {bob.items}""")
+#Printing Bob
+#bobsheet()	
+	
+###Define Battlers###
+class Battler(Character):
+	def __init__(self, name, position, items, attack, defense, health, magic, experience, level, playclass):
+		self.name= name
+		self.position= position
+		self.items= items
+		self.attack= attack
+		self.defense= defense
+		self.health= health
+		self.magic= magic
+		self.experience= experience
+		self.level= level
+		self.playclass= playclass
+#name, position, items, attack, defense, health, magic, experience, level, playclass
+###        ###          ###          ###         ###
+###Setters and Getters for Battlers###
+#Attack	
+	def attack(self):
+		return self.attack
+	def attack(self, value):
+		self.value= value
+#Defense	
+	def defense(self):
+		return self.defense
+	def defense(self, value):
+		self.value= value
+#Health
+	def health(self):
+		return self.health
+	def health(self, value):
+		self.value= value	
+#Magic
+	def magic(self):
+		return self.magic
+	def magic(self, value):
+		self.value= value	
+#Experience	
+	def experience(self):
+		return self.experience
+	def experience(self, value):
+		self.value= value
+#Level
+	def level(self):
+		return level.health
+	def level(self, value):
+		self.value= value	
+#Playclass
+	def playclass(self):
+		return self.playclass
+	def playclass(self, value):
+		self.value= value
+		
+###Start PC 'player'.###
+player= Battler(givename(), "Town", None, 1, 1, 1, 'none', 0, 1, giveclass())
+if player.playclass== 'A':
+	player.playclass='Warrior'
+	player.items= 'Sword'
+	player.attack= 5
+	player.defense= 10
+	player.health= 15
+	player.magic= 'None'
+elif player.playclass=='B':
+	player.playclass='Wizard'
+	player.items='Wand'
+	player.attack= 10
+	player.defense= 5
+	player.health= 10
+	player.magic= 'Fire'
+elif player.playclass=='C':
+	player.playclass='Cleric'
+	player.items='Relic'
+	player.attack= 4
+	player.defense= 10
+	player.health= 12
+	player.magic= 'Heal'
+elif player.playclass=='D':
+	player.playclass='Rogue'
+	player.items='Dagger' +' ' + 'Lockpick'
+	player.attack= 8
+	player.defense= 8
+	player.health= 10
+	player.magic= 'None'
+else:
+		None
+
+#Function to level up player#
+def levelup():
+		check=player.level
+		if player.experience<5:
+			player.level= 1
+		elif player.experience<10:
+			player.level= 2
+		elif player.experience<20:
+			player.level= 3
+		elif player.experience>20:
+			player.level= 4
+		else:
+			None
+		if player.level != check:
+			print("You leveled up!")
+		else:
+			None
+		
+#Function to call player statblock
+def playerstatblock():
```

Comment:
Make this a method in the class. Then don't assign self = player.

Reply:
I think doing so will generalize the ability to call the statblock.